install/tools/man/ipa-pkinit-manage.1

```diff
@@ -0,0 +1,34 @@
+.\"
+.\" Copyright (C) 2017  FreeIPA Contributors see COPYING for license
+.\"
+.TH "ipa-pkinit-manage" "1" "Jun 05 2017" "FreeIPA" "FreeIPA Manual Pages"
+.SH "NAME"
+ipa\-pkinit\-manage \- Enables or disables PKINIT
```

Comment:
The same phrasing works for `ipa-compat-manage`, `ipa-managed-entries` and `ipa-nis-manage`, so I don't see why it should not work here as well.